### ISSUE TITLE
host parameter 

### ISSUE THREAD

**Issue by xbgmsharp** [DavidMStraub/gramps-webapp#32]
To be able to deploy to the web, I had to add the host parameters to listen on all interfaces.

https://github.com/DavidMStraub/gramps-webapp/blob/master/gramps_webapp/__main__.py#L31

```
sed -i s'/(port/(host="0.0.0.0",port/'g gramps-webapp/gramps_webapp/__main__.py
```

**Comment by DavidMStraub** [comment]
Please don't do that, it uses flask's development server which is not meant for production, [see here ](https://flask.palletsprojects.com/en/1.1.x/tutorial/deploy/).

I should really add deployment instructions, but I use gunicorn:
```
gunicorn -b 0.0.0.0:$PORT gramps_webapp.wsgi:app
```

**Comment by xbgmsharp** [comment]
Thanks. Would love to see the web deployment instructions.
Will checkout gunicorn

**Comment by DavidMStraub** [comment]
Closed in favor of #34.
